### DISCUSSION TITLE
LLHLS ghost stream

### DISCUSSION THREAD

**Discussion by mvximvs** [OvenMediaLabs/OvenMediaEngine#1657] [Help]
Hi Community, 
With my team we had implemented LLHLS streaming. So we have the following architecture:

--- FLOW ---
`Camera(local) ==push RTSP==> Oven Media Engine (local) ==push RTMP==>  Oven Media Engine (cloud)  = LLHLS   Stream <=== Client`

*This is part of an IoT project, so when I say local is the edge of the organization,...

Everything was working good but we've started experiencing some connectivity problems on the local part of the organization and the oven media(local) wasn't able to continuing to push the stream.... but the oven media on the cloud says that the stream was still there... 

So users (client) just see the last image of the stream with a spinner... like a cache of the last frame...
Then, when the connection is reestablished, we push again the flow to the  oven media cloud instance.
But the cloud oven media says I already have a stream with that name (the ghost) you have to change the name of your stream before pushing.

We are using the last version of Oven Media => 0.16.6  in both instances, local and cloud.
But this problem was present since the 0.15.1.

Is this problem something that you have already experienced ?
If you need more details, just tell me and I will provide everything that I have.

Thanks in advance for your help

**Comment by getroot** [comment]
Is it working with [Camera <-- pull rtsp <-- OME] configuration?

Since OME does not support push rtsp, you need to check.

I think this issue can occur when camera and ome(local) are connected with tcp, but camera does not send any data. Try to query information about input stream using OME(local) API. It might help to look at bitrates, fps, etc.

Of course, it is also possible that it is a bug in OME. There might be a problem with rtsp provider module, so it can no longer receive incoming traffic. Are there any clues in the logs on camera side or ome?

We can get more information if we check the full ome log file and server.xml when the problem occurred.
